docs/devel/README.md

```diff
@@ -100,18 +100,18 @@ The following commands will execute:
   - ssh enabled
 
 ```bash
-dnf install podman -y
+dnf install podman -y && dnf clean all
 git clone https://gitlab.com/CentOS/automotive/sample-images.git
-cd sample-images/osbuild-manifests/
-dnf clean all
+git submodule update --init
```

Comment:
I would redirect here to the relevant link in automotive sig, in case it will change in the future

Reply:
Please ignore, already exist above